modules/statistics/statistics.js

```diff
@@ -238,10 +239,41 @@ Statistics.prototype.startRemoteStats = function(peerconnection) {
 
 Statistics.localStats = [];
 
-Statistics.startLocalStats = function(stream, callback) {
+Statistics.startLocalStats = function(track, callback) {
+    track.addEventListener(
```

Comment:
Do we need to add the listeners if audio levels are disabled?

Reply:
yep, because the AudioContext is always created.